### PR TITLE
Added pool, buoys, and gate models to world.

### PR DESCRIPTION
**Pull request by dcrossman**
Buoy and gate are not currently positioned correctly but easy to
change, as well as add more for each quadrant of pool.

This commit is included in the camera pull request so this one should probably be accepted first.

### CODE REVIEW

File: models/pool/model.config [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" ?>
+<model>
+    <name>Pool</name>
+    <version>1.0</version>
+    <sdf version="1.5">pool.sdf</sdf>
+    <author>
+        <name>Dustin Crossman</name>
+        <email>dustin.crossman@gmail.com</email>
+    </author>
+    <description>A model for the TRANDEC pool</description>
```
Comment (bkallaher) [inline models/pool/model.config]: Should be TRAN**S**DEC

File: worlds/underwater.world [hunk 2]
```diff
@@ -211,6 +215,26 @@
...
+      <pose>-5 -5 4 0 0 0</pose>
+      <uri>model://gate</uri>
+    </include>
+
+    <include>
+      <pose>0 3 4 0 0 0</pose>
+      <uri>model://buoys</uri>
+    </include>
+
+    <include>
```
Comment (bkallaher) [inline worlds/underwater.world]: Verify that the size and ordering is correct for the buoys. later also rename.

**Review by bkallaher — CHANGES_REQUESTED** [review]
Later remove Collada files as the models are simple and shouldn't require meshes

**Comment by dcrossman** [comment]
Pool and obstacle sizes are verified. Exact obstacle positions are eyeballed but should be close. Sub now spawns by side A dock.

File: worlds/underwater.world (hunk 2 above)
Comment (dcrossman) [reply worlds/underwater.world]: Renamed and size verified!

File: models/pool/model.config (hunk 1 above)
Comment (dcrossman) [reply models/pool/model.config]: Fixed

**Review by irwineffect — APPROVED** [review]
looks good!

**Review by bkallaher — APPROVED** [review]

**Comment by bkallaher** [comment]
I've made an issue concerning the collada to SDF conversion as it really isn't needed for this PR to work.

**Pull request merged**